src/api/model/vault/entry.ts

```diff
@@ -0,0 +1,5 @@
+export default abstract class Entry {
```

Comment:
Can this be done as an interface instead?

Reply:
Probably. But this vault thing is early proposal. We need more conversation. Consider this an internal API atm.